build.gradle

```diff
@@ -1,42 +1,25 @@
 buildscript {
-    ext.kotlinVersion = "1.6.21"
     ext.compileVersion = 33
-    ext.buildToolsVersion = "33.0.2"
     ext.minVersion = 14
-    ext.billingVersion = "5.1.0"
-    ext.lifecycleVersion = "2.5.0"
-    ext.testLibrariesVersion = "1.4.0"
-    ext.testJUnitVersion  = "1.1.3"
-    ext.robolectricVersion = "4.10.3"
-    ext.mockkVersion = "1.12.8"
-    ext.assertJVersion = "3.22.0"
-    ext.annotationVersion = "1.3.0"
-    ext.androidxCoreVersion = "1.8.0"
-    ext.mockwebserverVersion = "4.2.0"
-    ext.tinkVersion = "1.8.0"
-    ext.detektVersion = "1.23.0"
-    ext.coroutinesVersion = "1.6.4"
-
-    repositories {
-        mavenCentral()
-        google()
-    }
-    dependencies {
-        classpath "com.vanniktech:gradle-maven-publish-plugin:0.22.0"
-        classpath 'com.android.tools.build:gradle:8.0.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath "org.jetbrains.dokka:dokka-gradle-plugin:1.8.10"
-        classpath "org.jetbrains.kotlinx:kover-gradle-plugin:0.7.0"
-    }
 }
 
 plugins {
-    id "io.gitlab.arturbosch.detekt" version "$detektVersion"
-    id "com.savvasdalkitsis.module-dependency-graph" version "0.9"
+    alias libs.plugins.mavenPublish apply false
+    alias libs.plugins.android.application apply false
+    alias libs.plugins.android.library apply false
```

Comment:
So now we are adding both the application and library plugins here and applying them on the respective modules each. Is it necessary to specify both here?

Reply:
Yes. Think this is the equivalent of adding the AGP plugin. Removing either plugin gives the same error as removing the parcelize (The request for this plugin could not be satisfied because the plugin is already on the classpath with an unknown version, so compatibility cannot be checked.). Removing `application` gives the issue in the apps, I am getting it in `api-tester`, and removing the library one gives it in `common`.

I think these plugins need to be applied both in the base and in the module's build.gradle. I guess it makes sense for the android plugins, because at the end of the day the need to build an android project, and this is the build.gradle of the project, but I am still not sure about the parcelize one.